restli-int-test-server/src/main/java/com/linkedin/restli/examples/greetings/server/defaults/FieldFillInDefaultResources.java

```diff
@@ -0,0 +1,19 @@
+package com.linkedin.restli.examples.greetings.server.defaults;
+
+import com.linkedin.restli.examples.defaults.api.HighLevelRecordWithDefault;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.resources.CollectionResourceTemplate;
+
+
+@RestLiCollection(name = "fillInDefaults", namespace = "com.linkedin.restli.examples.defaults.api")
+public class FieldFillInDefaultResources extends CollectionResourceTemplate<Long, HighLevelRecordWithDefault>
+{
+  @Override
+  public HighLevelRecordWithDefault get(Long keyId)
```

Comment:
If you change this to a create with return entity, then you can controll the data being returned (by sending it for create) and thus can test multiple scenarios

Reply:
ok